src/Spec/Swagger2.php

```diff
@@ -238,6 +238,7 @@ public function getMethods($service)
                                 $output['parameters']['query'][] = $param;
                                 break;
                             case 'formData':
+                                $param['default'] = (is_array($param['default']) || $param['default'] instanceof stdClass) ? json_encode($param['default']) : $param['default'];
```

Comment:
Let's update the logic on line 223 to check `stdClass` as well instead of adding another check here

Reply:
done 👍  thanks